### DISCUSSION TITLE
EMQX Docker on ECS Error

### DISCUSSION THREAD

**Discussion by thehellmaker** [emqx/emqx#11828] [General]
Hi, 
I am trying to run EMQX Docker on ECS attached to and EC2 instance with AMI tuned for high performance according to ECS docs. Trying to run the docker from CLI using docker run works well. But when i am trying to run the same from ECS task it fails with this wierd error 

whoami: cannot find name for user ID 1000: No such file or directory

I have tried the same on Ubuntu, Amazon Linux2 both and they both have the same issue.

**Comment by ieQu1** [comment]
This PR may be the solution to this problem: https://github.com/emqx/emqx/pull/11770/files 
I'll check if we can merge it.

**Reply by ieQu1** [reply]
P.S. The PR has been merged, so hopefully this issue will be resolved in the next release.

**Reply by thehellmaker** [reply]
Thanks a lot for the quick fix. There was a release yesterday 5.3-elixir on docker hub https://hub.docker.com/layers/emqx/emqx/5.3-elixir/images/sha256-7e2ee58cb4891382de45fe2474a8ab03e82d020b2bbbe80e0e0e4da6f11ca3f2?context=explore and this still has the same issue. How do i check if this fix was a part of that docker release?

**Reply by savonarola** [reply]
Hi! The releases can be found here: https://github.com/emqx/emqx/releases, e.g. https://github.com/emqx/emqx/releases/tag/v5.3.1
Unfortunately, the fix didn't get to the release branch which was already formed (and to the release itself). It will get to the next release.
